docs/whats-new/2020-06.md

```diff
@@ -0,0 +1,136 @@
+
+## .NET Framework
+
+### New articles
+
+- [ComNetOS class](/dotnet/framework/additional-apis/system.net.comnetos)
+  - Add System.Net APIs (1 of 3)
+- [ExceptionHelper class](/dotnet/framework/additional-apis/system.net.exceptionhelper)
+  - Add System.Net APIs (1 of 3)
+- [HttpStatusDescription class](/dotnet/framework/additional-apis/system.net.httpstatusdescription)
+  - Add System.Net APIs (1 of 3)
+- [Logging class](/dotnet/framework/additional-apis/system.net.logging)
+  - Add System.Net APIs (1 of 3)
+- [MailAddressParser class](/dotnet/framework/additional-apis/system.net.mail.mailaddressparser)
+  - Add System.Net APIs (2 of 3)
+- [QuotedPairReader class](/dotnet/framework/additional-apis/system.net.mail.quotedpairreader)
+  - Add System.Net APIs (2 of 3)
+- [MailBnfHelper class](/dotnet/framework/additional-apis/system.net.mime.mailbnfhelper)
+  - Add System.Net APIs (3 of 3)
+- [ServicePointManager.CloseConnectionGroups method](/dotnet/framework/additional-apis/system.net.servicepointmanager.closeconnectiongroups)
+  - Add System.Net APIs (2 of 3)
+- [UnsafeNclNativeMethods class](/dotnet/framework/additional-apis/system.net.unsafenclnativemethods)
+  - Add System.Net APIs (2 of 3)
+- [WebHeaderCollection.AddInternal method](/dotnet/framework/additional-apis/system.net.webheadercollection.addinternal)
+  - Add System.Net APIs (2 of 3)
+
+## F# language
+
+### Updated articles
+
+- [Keyword Reference](/dotnet/fsharp/language-reference/keyword-reference)
+  - Fix default checksum algorithm
+
+## ML.NET
+
+### Updated articles
+
+- [The ML.NET CLI command reference](/dotnet/machine-learning/reference/ml-net-cli-reference)
+  - Update mlnet cli docs
+
+## .NET for Apache Spark
+
+### New articles
+
+- [Use broadcast variables in .NET for Apache Spark](/dotnet/spark/how-to-guides/broadcast-guide) - Added docs for broadcast and UDF guide
+- [Create user-defined functions (UDF) in .NET for Apache Spark](/dotnet/spark/how-to-guides/udf-guide) - Added docs for broadcast and UDF guide
+
+## .NET
+
+### New articles
+
+- [Implement a DisposeAsync method](/dotnet/standard/garbage-collection/implementing-disposeasync) - Implement the async dispose pattern
+
+## Community contributors
+
+The following people contributed to the .NET docs in June 2020. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [NextTurn](https://github.com/NextTurn) - Next Turn (52)
+- [DennisLee-DennisLee](https://github.com/DennisLee-DennisLee) - Dennis Lee (32)
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (14)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (8)
+- [MarcoRossignoli](https://github.com/MarcoRossignoli) - Marco Rossignoli (4)
+- [nschonni](https://github.com/nschonni) - Nick Schonning (3)
+- [alfredmyers](https://github.com/alfredmyers) - Alfred Myers (2)
+- [ardalis](https://github.com/ardalis) - Steve Smith (2)

```

Comment:
vendor

Reply:
@MaximRouiller Make sure you have the latest `PageGenerator` changes. There was a recent change that removes these folks from the contributors section.